addons/mail/static/src/components/follower/follower.xml

```diff
@@ -4,15 +4,15 @@
     <t t-name="mail.Follower" owl="1">
         <div class="o_Follower">
             <t t-if="follower">
-                <a class="o_Follower_details" t-att-class="{ 'o-inactive': !follower.isActive }" href="#" t-on-click="_onClickDetails">
-                    <img class="o_Follower_avatar" t-attf-src="/web/image/{{ follower.resModel }}/{{ follower.resId }}/image_128" alt="Avatar"/>
+                <a role="menuitem" class="o_Follower_details" t-att-class="{ 'o-inactive': !follower.isActive }" href="#" t-on-click="_onClickDetails">
+                    <img class="o_Follower_avatar" t-attf-src="/web/image/{{ follower.resModel }}/{{ follower.resId }}/image_128" alt=""/>
```

Comment:
why remove alt?

Reply:
Alt's are intended to describe images when they add useful information. In this case, it adds noise, because what we need to know is the follower name, but this adds an extra "abatar" message before every follower name.